index.bs

```diff
@@ -114,10 +114,14 @@ interface AnimationWorkletGlobalScope : WorkletGlobalScope {
     Note: This is how the class should look.
     <pre class='lang-javascript'>
         class FooAnimator {
-            static get inputProperties() { return ['--foo']; }
-            static get outputProperties() { return ['transform']; }
-
-            animate(root, children, timeline) {
+            static get elements() {
```

Comment:
Non-actionable comment: 'elements()' seems misleading to me here, it makes it seem like the element should be named 'foo' or 'bar' rather than the element has a slot name of 'foo' or 'bar'.